fsspec/spec.py

```diff
@@ -405,11 +405,17 @@ def size(self, path):
 
     def isdir(self, path):
         """Is this entry directory-like?"""
-        return self.info(path)['type'] == 'directory'
+        try:
```

Comment:
Quite right, this better reflects `os.path`.
Care to make a test of the two nonexistent cases, at least for the LocalFileSystem?

Reply:
Will do

Reply:
Done!